tensorflow/contrib/model_pruning/README.md

```diff
@@ -133,7 +133,8 @@ For now, it is assumed that the underlying hardware platform will provide mechan
 
 ## Example: Pruning and training deep CNNs on the cifar10 dataset <a name="example"></a>
 
-Please see https://www.tensorflow.org/tutorials/deep_cnn for details on neural
+Please see [Advanced Convolutional Neural Networks
+](https://www.tensorflow.org/tutorials/images/deep_cnn) for details on neural
```

Comment:
Could you please move the closing bracket to the previous line?
```
Please see [Advanced Convolutional Neural Networks]
(https://www.tensorflow.org/tutorials/images/deep_cnn) for details on neural
```

Reply:
Moved things around, see dd0a1b2